account_payment_order/models/account_payment_order.py

```diff
@@ -106,6 +105,18 @@ def domain_journal_id(self):
         readonly=True)
     description = fields.Char()
 
+    # [bt#5551] From v12
+    @api.depends('payment_mode_id')
+    def _compute_allowed_journal_ids(self):
+        for record in self:
+            if record.payment_mode_id.bank_account_link == 'fixed':
+                record.allowed_journal_ids = (
+                    record.payment_mode_id.fixed_journal_id)
+                print("")
```

Comment:
Print commands should be replaced with logger ones

Reply:
And empty ones removed :)